lib/cc/analyzer/issue_validations/location_format_validation.rb

```diff
@@ -2,42 +2,50 @@ module CC
   module Analyzer
     module IssueValidations
       class LocationFormatValidation < Validation
-        def valid?
-          if location["lines"]
-            valid_lines?(location["lines"])
-          elsif location["positions"]
-            valid_positions?(location["positions"])
-          else
-            false
+        class Validator
```

Comment:
Is this class mostly just a refactor? I don't see a lot of value in it and might avoid muddying the diff if not directly related to the change.